continunet/tests/test_network_unet.py

```diff
@@ -0,0 +1,91 @@
+"""Tests for the UNet model."""
+
+import pytest
+
+from continunet.network.unet import Unet
+
+
+class TestUnet:
+    """Tests for the UNet model."""
+
+    model = Unet
+
+    def test_build_model(self, input_shape):
+        """Test the compile_model method"""
+
+        test_model = self.model(input_shape).build_model()
+
+        assert test_model is not None
+        assert test_model.input_shape == (None, input_shape[0], input_shape[1], input_shape[2])
```

Comment:
```suggestion
        assert test_model.input_shape == (None, *input_shape)
```
should work to destructure